src/components/[guild]/activity/constants.ts

```diff
@@ -288,3 +293,27 @@ export const activityLogActionIcons: Record<
     color: "blue.400",
   },
 }
+
+export const HIDDEN_ACTIONS: ACTION[] = [
+  ACTION.UpdateUrlName,
+  ACTION.UpdateLogoOrTitle,
+  ACTION.UpdateDescription,
+  ACTION.UpdateLogic,
+  ACTION.UpdateTheme,
+]
+
+export const USER_ACTIONS: ACTION[] = [
+  ACTION.ClickJoinOnWeb,
+  ACTION.ClickJoinOnPlatform,
+  ACTION.JoinGuild,
+  ACTION.LeaveGuild,
+  ACTION.UserStatusUpdate,
+  ACTION.GetRole,
+  ACTION.LoseRole,
+  ACTION.GetReward,
+  ACTION.LoseReward,
```

Comment:
There are a couple of missing action here: `KickFromGuild`, `SendReward`, `RevokeReward` (these happen automatically)